HPXMLtoOpenStudio/resources/hvac.rb

```diff
@@ -3546,25 +3555,30 @@ def self.calc_sequential_load_fractions(load_fraction, remaining_fraction, avail
     return sequential_load_fracs
   end
 
-  def self.get_sequential_load_schedule(model, fractions)
+  def self.get_sequential_load_schedule(model, fractions, power_outage_periods = [])
     values = fractions.map { |f| f > 1 ? 1.0 : f.round(5) }
 
-    if values.uniq.length == 1
+    if values.uniq.length == 1 && power_outage_periods.empty?
       s = OpenStudio::Model::ScheduleConstant.new(model)
       s.setValue(values[0])
```

Comment:
Why aren't you using your new ScheduleRulesetConstant class for this situation?

Reply:
Using it now.